x/epochstorage/keeper/params.go

```diff
@@ -107,12 +107,23 @@ func (k Keeper) BlocksToSave(ctx sdk.Context, block uint64) (res uint64, erro er
 
 func (k Keeper) BlockInEpoch(ctx sdk.Context, block uint64) (res uint64, err error) {
 	// get epochBlocks directly because we also need an epoch start on the current grid and when fixation was saved is an epoch start
-	fixtedParams, err := k.GetFixatedParamsForBlock(ctx, string(types.KeyEpochBlocks), block)
-	var blocksCycle uint64
-	utils.Deserialize(fixtedParams.Parameter, &blocksCycle)
-	epochStartInGrid := fixtedParams.FixationBlock // fixation block is always <= block
+	fixatedParams, err := k.GetFixatedParamsForBlock(ctx, string(types.KeyEpochBlocks), block)
+	if err != nil {
+		return 0, err
+	}
+
+	var epochBlocks uint64
+	utils.Deserialize(fixatedParams.Parameter, &epochBlocks)
+
+	if epochBlocks == 0 {
```

Comment:
we don't need to validate a param on every action if we have validate basic that protects from 0

Reply:
just to clarify Omer's comment, the check for 0 already exists in validateEpochBlocks()

Reply:
Talked to @omerlavanet about this. Decided to leave it as it is.